packages/mui-base/src/SliderUnstyled/useSlider.ts

```diff
@@ -576,10 +567,6 @@ export default function useSlider(props: UseSliderProps) {
   const createHandleMouseDown =
     (otherHandlers: Record<string, React.EventHandler<any>>) =>
     (event: React.MouseEvent<HTMLSpanElement, MouseEvent>) => {
-      if (onMouseDown) {
-        onMouseDown(event);
-      }
-
       // Only handle left clicks
```

Comment:
The external handler should be called:

```suggestion
      otherHandlers.onMouseDown?.(event);
      if (event.defaultPrevented) {
        return;
      }
      
      // Only handle left clicks
```